manual/manual/html_processing/scss/manual.scss

```diff
@@ -0,0 +1,292 @@
+}
+p,ul{
+    margin-top:.5em;
+    margin-bottom:1em
+}
+ul{
+    list-style-position:outside
+}
+ul>li{
+    margin-left:22px
+}
+li>:first-child{
+    margin-top:0
+}
+.left{
+    text-align:left
+}
+.right{
+    text-align:right
+}
+a{
+    text-decoration:none;
+    color:#92370a
+}
+a:hover{
+    box-shadow:0 1px 0 0 #92370a
+}
+:target{
+    background-color:rgba(255,215,181,.3)!important;
+    box-shadow:0 0 0 1px rgba(255,215,181,.8)!important;
+    border-radius:1px
+}
+:hover>a.section-anchor{
+    visibility:visible
+}
+a.section-anchor:before{
+    content:"#"
+}
+a.section-anchor:hover{
+    box-shadow:none;
+    text-decoration:none;
+    color:#555
+}
+a.section-anchor{
+    visibility:hidden;
+    position:absolute;
+    margin-left:-1.3em;
+    font-weight:400;
+    font-style:normal;
+    padding-right:.4em;
+    padding-left:.4em;
+    color:#d5d5d5
+}
+.h10,.h7,.h8,.h9,h1,h2,h3,h4,h5,h6{
+    font-family:"Fira Sans",Helvetica,Arial,sans-serif;
+    font-weight:400;
+    margin:.5em 0 .5em 0;
+    padding-top:.1em;
+    line-height:1.2;
+    overflow-wrap:break-word
+}
+h1{
+    font-weight:500;
+    font-size:2.441em;
+    margin-top:1.214em
+}
+h1{
+    font-weight:500;
+    font-size:1.953em;
+    box-shadow:0 1px 0 0 #ddd
+}
+h2{
+    font-size:1.563em
+}
+h3{
+    font-size:1.25em
+}
+h1 code{
+    font-size:inherit;
+    font-weight:inherit
+}
+h2 code{
+    font-size:inherit;
+    font-weight:inherit
+}
+h3 code{
+    font-size:inherit;
+    font-weight:inherit
+}
+h3 code{
+    font-size:inherit;
+    font-weight:inherit
+}
+h4{
+    font-size:1.12em
+}
+.ocaml,.pre,code,pre,tt{
+    font-family:"Fira Mono",courier;
+    font-weight:400
+}
+.pre,pre{
+    border-left:4px solid #e69c7f;
+    overflow-x:auto;
+    padding-left:1ex
+}
+li code,p code{
+    background-color:#f6f8fa;
+    color:#0d2b3e;
+    border-radius:3px;
+    padding:0 .3ex
+}
+.pre .code,.pre.code,pre code{
+    background-color:inherit
+}
+p a>code{
+    color:#92370a}
+.pre code.ocaml,.pre.code.ocaml,pre code.ocaml{
+    font-size:.893rem}
+.keyword,.ocamlkeyword{
+    font-weight:500}
+section+section{
+    margin-top:25px}
+ul.itemize{
+    list-style:none}
+ul.itemize li::before{
+    content:"▶";
+    color:$logocolor;
+    margin-right:4px;
+    margin-left:-1em}
+
+/* Table of Contents in the Left-hand sidebar */
+nav.toc{
+    @include nav-toc;
+    &.brand{
+	@include brand;
+    }
+    .toc_title{
+	display:block;
+	margin:.5em 0 1.414em}
+/* .toc_title a{ */
+/*     color:#777; */
+/*     font-size:1em; */
+/*     line-height:1.2; */
+    /*     font-weight:500} */
+
+}
+.tableau {
+    table {
+	border-collapse: collapse;
+    }
+    td {
+	background:linear-gradient(to left,#fff 0,#ede8e5 100%);
+	border:1px solid #ccc;
+	padding-left:3px;
+	padding-right:3px;
+    }
+}
+
+/* Mobile */
+@media only screen and (max-width:95ex){
+    .content{
+	margin:auto;
+	padding:2em
+    }
+    nav.toc{
+	@include nav-toc-mobile;}
+}
```

Comment:
In this mode, it is impractical to go to the next page when reading linearly: the table of the content is at the top of the page and there is no navigation element at the bottom of the page.

Reply:
what do you mean by next page? do you mean next section? I agree this is not very practical. I'm not a big user of the mobile interface, what would users expect? I guess, a button to open the TOC ?

Reply:
I meant the next section or page in the linear order of the manual. Maybe a link to the TOC anchor, previous and next pages? In fact, the next and previous links could be useful in the normal mode. Currently going to the next page takes between two and three actions away: either a click to the OCaml manual page, a search in the Toc, and then a click if we cross a part boundary; or it requires to go back to the top of page, and click on the next section. Having an one click shortcut might be worthwhile?

Reply:
apparently, `odoc` doesn't handle this either. I have some ideas but this would require some more javascript.

Reply:
my opinion was that reading the whole manual in a linear way was not useful, but maybe I'm wrong.
To me, the "next/previous" links had no intelligent meaning: when you are at the end of a chapter, you have no clue about what you will discover by clicking on these links, that's why I removed them.

Reply:
what one could do is to replace "next/previous" by the title of the chapters they link to

Reply:
The manual is written to be readable in a linear way .
If we could have link with the names of the previous and next chapter (at the bottom to not repeat the TOC) that would be ideal in my opinion.

Reply:
great, I'll try to do this then

Reply:
done

Reply:
This works very well, thanks!